### DISCUSSION TITLE
Question about upload file and send it in email

### DISCUSSION THREAD

**Discussion by AABehzad** [PHPMailer/PHPMailer#2349] [General]
Hi friends , I have a Question about upload file and send it in email ,I get user's file in form Do I have to upload the file to the site after receiving it from the user ??!! or can it be sent directly to the email? ???!

**Comment by aleksandrmelnikov** [comment]
You have to store the uploaded file from the form.

You can follow this example.
https://www.github.com/PHPMailer/PHPMailer/tree/master/examples%2Fsend_file_upload.phps

**Reply by AABehzad** [reply]
thanks
